scripts/Expand-ExchangeFiles.ps1

```diff
@@ -14,7 +14,7 @@ try {
     $pass = ConvertTo-SecureString $Password -AsPlainText -Force
     $cred = New-Object System.Management.Automation.PSCredential -ArgumentList $UserName,$pass
         
-    Invoke-Command -ScriptBlock {Start-Process cmd.exe '/c c:\exchinstall\Exchange2013-x64.exe /extract:c:\exchinstall /passive' -NoNewWindow -Wait} -ComputerName $ComputerName -Credential $cred
+    Invoke-Command -ScriptBlock {Start-Process cmd.exe '/c c:\Exchangeinstall\Exchange2013-x64.exe /extract:c:\Exchangeinstall\bin /passive' -NoNewWindow -Wait} -ComputerName $ComputerName -Credential $cred
```

Comment:
The newer template is completely independent of the old one, and both should work. I reuse some existing scripts but where I needed new functionality I wrote new scripts such as 

- Expand-ExchangeFiles.ps1 (old QS)
- Expand-Exchange.ps1 ( new QS)
- Install-ExchPreReq (old QS)
- Install-ExchPrerequisites (new QS)